modular_skyrat/code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1,3 +1,8 @@
+/datum/reagent/medicine/mine_salve/on_mob_metabolize(mob/living/M) //modularisation for miners salve painkiller.
```

Comment:
and on_mob_end_metabolize?

Reply:
already handled in medicine_reagents.dm (the unmodularised one)